app/api/v1/blueprints/offices.py

```diff
@@ -10,7 +10,13 @@
 @login_required
 def create_office():
     data = request.get_json()
-    
+    if not data:
+        response = {
+            'status': 400,
+            'error': 'Request is empty'
```

Comment:
The **error** message should let users know what kind of inputs are required. Let me know when you implement this.

Reply:
Ok I will.

Reply:
I have updated the endpoints.